### PR TITLE
[Port-TG] Add defines for byond-tracy support 

### PR DESCRIPTION
**Pull request by Phoenix4O4**
Not intended for use in production, and we do not ship a copy of byond-tracy. It is extremely easy to compile yourself, but if you're someone interesting enough to play around with this then let me know and I can just give you a build.
I'm going to be using this for init profiling.
image

**Port** [Add defines for byond-tracy support #70931](https://github.com/tgstation/tgstation/pull/70931)

## Why It's Good For The Game
Metrics

## Changelog
Developer stuff

### CODE REVIEW

File: tools/tracy_replay/replay.py
Comment (DeltaFire15) [inline tools/tracy_replay/replay.py:1]: Shouldn't this file not be in this PR if you are only adding the integration and not the tool itself (like the ported PR)?

File: code/_compile_options.dm [hunk 1]
```diff
@@ -32,6 +32,13 @@
 
 #endif //ifdef REFERENCE_TRACKING
 
+// If this is uncommented, we do a single run though of the game setup and tear down process with unit tests in between
+// #define UNIT_TESTS
```
Comment (DeltaFire15) [inline code/_compile_options.dm:36]: duplicate define located in l. 53. Remove the old one if you want to move it here.

File: tools/tracy_replay/replay.py
Comment (4310V343k) [reply tools/tracy_replay/replay.py:1]: replay.py is already [deprecated](https://github.com/ParadiseSS13/Paradise/blob/bf5a82365a3bfb42628a4b6670d867248956eeb3/tools/tracy_replay/README.MD) in favour of https://github.com/AffectedArc07/ParaTracyReplay, so yeah, it shouldn't be

**Comment by SandPoot** [comment]
I'm not bothering with this anymore, i compiled the file, set up defines and all and still nothing.

**Pull request closed without merging**